factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/registry/AbstractSchemaRegistry.java

```diff
@@ -87,8 +88,14 @@ public void fetchInitial() {
     synchronized (mutex) {
       Stopwatch sw = Stopwatch.createStarted();
       log.info("Registry update started");
-      indexFetcher.fetchIndex().ifPresent(this::process);
-      log.info("Registry update finished in {}ms", sw.stop().elapsed(TimeUnit.MILLISECONDS));
+      try {
+        indexFetcher.fetchIndex().ifPresent(this::process);
+        log.info("Registry update finished in {}ms", sw.stop().elapsed(TimeUnit.MILLISECONDS));
+      } catch (Throwable e) {
+        log.warn("While fetching initial state of registry", e);
```

Comment:
I wonder if we should handle this differently depending on how `persistentSchemaRegistry` is set. 🤔
What I mean is if theres no persistent schema registry booting an empty FC also makes no sense right? 🤔

Reply:
well. that was the initial assumption, which turned out to be impractical, hence this ticket... :thinking: